.travis.yml

```diff
@@ -2,15 +2,21 @@ language: ruby
 
 cache: bundler
 
-rvm:
-  - 2.2.2
+before_script:
+  - bundle exec danger
+
+env:
+  - ELASTICSEARCH=https://download.elastic.co/elasticsearch/release/org/elasticsearch/distribution/deb/elasticsearch/2.4.4/elasticsearch-2.4.4.deb
+  - ELASTICSEARCH=https://download.elasticsearch.org/elasticsearch/release/org/elasticsearch/distribution/deb/elasticsearch/2.1.1/elasticsearch-2.1.1.deb
+  - ELASTICSEARCH=https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-1.7.6.deb
+  - ELASTICSEARCH=https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-1.5.2.deb
```

Comment:
I wasn't aware of this Travis feature (doc [here](https://docs.travis-ci.com/user/environment-variables/#Defining-Multiple-Variables-per-Item)) - just to clarify, this will trigger a build per ENV var when a var has been defined multiple times. :+1: